### ISSUE TITLE
Missing context when opening Jupyter notebooks in VS Code

### ISSUE THREAD

**Issue by Exaphis** [TabbyML/tabby#788]
**Describe the bug**
Tabby only recognizes contents from the currently edited cell of a Jupyter notebook as context. Since cells build on each other, missing prior cells omits crucial information.

With a notebook containing two cells:
```python
var_6f17 = ''
var_ndd9 = ''
var_odrv = ''
```
```python
# Print all previous variables
```

Logging the prefix and suffix in the Tabby server shows that the prior cell contents are missing:
```
PREFIX: # Print all previous variables
, SUFFIX: Some("")
PROMPT:  <PRE># Print all previous variables
 <SUF>
 <MID>
```

GitHub Copilot is able to complete the prompt with the correct variable names, which shows that it does know about the previous cell:
```python
print(var_6f17)
print(var_ndd9)
print(var_odrv)
```

**Information about your version**
Please provide output of `tabby --version`
`tabby 0.6.0-dev`

**Comment by icycodes** [comment]
Thanks for reporting this bug. I will address and fix it in the upcoming release

**Comment by icycodes** [comment]
Released in Tabby IDE plugins version 1.2.0.
